README.md

```diff
@@ -467,6 +467,12 @@ You can visit `http://localhost:8080/api/graph/playground` to access a GraphQL p
 { "Authorization":"Local {\"favorLocalAuth\":true}"}
 ```
 
+Additionally, you can add EUA job codes, for use when querying endpoints such as `systemIntake` that require them. The syntax is:
```

Comment:
@downeyn-cms @ClayBenson94 Does this wording look good?

Reply:
Maybe something like

```suggestion
Additionally, you can add a header that defines EUA job codes that will be used when querying endpoints such as `systemIntake` that require them. The syntax is:
```

Reply:
How's the wording as of https://github.com/CMSgov/easi-app/pull/1391/commits/8d912e86e69c4a25e295294a35d33a2f3c32aa36 look?

Reply:
Perfect!